templates/corporate/upgrade.html

```diff
@@ -24,6 +24,16 @@ <h1>🚀 {% trans %}Upgrade to {{ plan }}{% endtrans %}</h1>
                 </div>
                 {% endif %}
 
+                {% if is_demo_organization %}
+                <div class="demo-organization-warning">
+                    Demo organizations cannot be upgraded to Zulip Cloud Standard. Please start by
```

Comment:
I propose this: "Demo organizations cannot be directly upgraded to a paid plan." 

(1) "directly" makes clear you can do it, just you have the do something first.
(2) Avoid mentioning the current plan name, since it's an issue regardless of what target plan.

Reply:
applied this tweak after checking with Alya.